packages/core/elasticsearch/core-elasticsearch-server-internal/src/elasticsearch_config.ts

```diff
@@ -37,6 +37,8 @@ export const configSchema = schema.object({
     defaultValue: 'http://localhost:9200',
   }),
   maxSockets: schema.number({ defaultValue: Infinity, min: 1 }),
+  maxIdleSockets: schema.number({ defaultValue: 256, min: 1 }),
+  idleSocketTimeout: schema.duration({ defaultValue: '9m' }),
```

Comment:
the  services forwarder has a timeout of 10minutes. But a race condition is possible where the proxy sends a close but there's some delay in the close packet reaching Kibana and at the same time Kibana might start a new request. So to be on the safe side we have a margin of 1min so that Kibana would close any idle sockets 1 minute before the proxy might try to close them.

<img width="554" alt="Screenshot 2022-07-28 at 11 16 26" src="https://user-images.githubusercontent.com/1281646/192645424-d7e0ab0b-202c-407c-9d17-cc5e22ce56df.png">